showItemPageBE/src/main/java/com/talentpath/shamazin/showItemPage/services/QuestionService.java

```diff
@@ -13,4 +13,14 @@
     QuestionRepository questionDao;
 
     public List<Question> getAllQuestions() {return questionDao.findAll();}
+
+    public Question getQuestionById(Integer qId){
+        return questionDao.findById(qId).get();

```

Comment:
I'm not super experienced with JPA, so I'm just curious what is the extra .get() getting from .findById(qId)? Does it not need some sort of argument to return a Question object?

Also, minor nitpick, I think we can put this all onto one line of code like everywhere else in this file, just for cleanliness's sake (unless there's a reason why it's put onto three lines of code).

Reply:
Ah yeah I have my code collapsed so it shows up on just one line, I can put everything on just one line though

Reply:
<img width="1440" alt="Screen Shot 2021-03-11 at 2 13 29 PM" src="https://user-images.githubusercontent.com/57366039/110841512-f6e53b00-8273-11eb-8ab5-c0acf7af165d.png">

Reply:
This is what happens when you don't include .get(), doesn't provide the correct type

Reply:
Oh, interesting, did not know that haha. Yeah, was just curious.
Yeah, everything still LGTM 👍🏻

Reply:
Dope thanks for reviewing! Also pushed up the change so that method is on one line

Reply:
Squeaky clean 🧼